walletunlocker/service_test.go

```diff
@@ -472,7 +504,20 @@ func TestChangeWalletPassword(t *testing.T) {
 			t.Fatalf("expected to receive password %x, got %x",
 				testPassword, unlockMsg.Passphrase)
 		}
+
+		// We'll now close the done channel to unlock the service to be
+		// able to accept another request.
```

Comment:
from your comment on the pr it seems the unlocker can only process one request, but several comments in the code refer to processing more requests. is it the case that the wallet unlocker _could_ process more requests, but we only allow one due to the `defer cancel()` in `waitForWalletPassword`?